templates/user_data.sh

```diff
@@ -1,9 +1,16 @@
 #!/bin/bash
-OS='${operating_system}'
-
-if [ "$${OS:0:6}" = "centos" ] || [ "$${OS:0:4}" = "rhel" ]; then
+distro=$(lsb_release -i -s | tr A-Z a-z)
+case $distro in
+  ubuntu|debian)
+    export DEBIAN_FRONTEND=noninteractive
+    apt update -q
+    apt install -qy systemd-timesyncd
+    apt autoremove -qy
+    ;;
+  centos|rhel)
```

Comment:
These values need to be verified.  (`redhat` maybe?)

Reply:
It's `RedHatEnterprise`, but lsb_release may not be preinstalled (I haven't confirmed this, my testing is from using `dnf install -y redhat-lsb-core` within `docker run -it --rm --name test registry.access.redhat.com/ubi8/ubi:8.1 bash`

Reply:
Using the same command from the `centos:8` image, I get `CentOS`.

The `tr` command used here lower-cases all of these values for the case statement.